spring-cloud-starter-stream-processor-header-enricher/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Seems for me we don't need this extra `.gitignore`.
The root one should be enough.